x-pack/plugins/transform/public/app/sections/create_transform/components/wizard/_wizard.scss

```diff
@@ -8,22 +8,11 @@
   }
 }
 
-.transform__stepDefineForm {
-  align-items: flex-start;
-}
-
-.transform__stepDefineFormLeftColumn {
-  min-width: 420px;
-  border-right: 1px solid $euiColorLightShade;
-}
-
 /*
-This is an override to replicate the previous full-page-width of the transforms creation wizard
-when it was in use within the ML plugin. The Kibana management section limits a max-width to 1200px
-which is a bit narrow for the two column layout of the transform wizard. We might revisit this for
-future versions to blend in more with the overall design of the Kibana management section.
-The management section's navigation width is 192px + 24px right margin
+This ensures the wizard goes full page width, and that the data grid in the page does not
+cause the body of the wizard page to overflow into the side navigation of the Kibana
+Stack Management page on resize.
 */
-.mgtPage__body--transformWizard {
-  max-width: calc(100% - 216px);
+.transform__wizardBody {
+  max-width: calc(100% - 16px);
```

Comment:
I've not tried this out, but at minimum this should be a variable value `$euiSize`